init.d/urandom.in

```diff
@@ -24,35 +21,33 @@ depend()
 
 save_seed()
 {
+	local psz=1
+
+	if [ -e /proc/sys/kernel/random/poolsize ]; then
```

Comment:
Does this file exist on FreeBSD? I would suspect not, but dunno.

Reply:
I don't know either, so I'm just keeping the code to be sure.

Reply:
I just booted up a FreeBSD box:

```
[root@devil ~]# cat /proc/sys/kernel/random/poolsize
cat: /proc/sys/kernel/random/poolsize: No such file or directory
[root@devil ~]# ls /proc
[root@devil ~]# ls /proc/sys
ls: /proc/sys: No such file or directory
[root@devil ~]# uname -a
FreeBSD devil 13.0-RELEASE FreeBSD 13.0-RELEASE #4 releng/13.0-n244733-ea31abc261f-dirty: Fri Apr 30 08:34:57 UTC 2021     root@devil:/usr/obj/usr/src/amd64.amd64/sys/GENERIC  amd64
```

So you can get rid of this stanza.